virttest/shared/cfg/base.cfg

```diff
@@ -1087,3 +1087,7 @@ uuid_dimm = ""
 
 # Set image_extent_size_hint, displayed in qemu-img info extend_size_hint field.
 #image_extent_size_hint = 1G #To set extent_size_hint to 1G
+
+
+# The parameter required_virtio_win is used to specify a range of virtio-win iso.
```

Comment:
```suggestion
# Set required_virtio_win, specify a range of virtio-win iso version (optional)
```

Reply:
@leidwang Sorry, after a second consideration, maybe better move to Windows.cfg, or where cdrom_virtio primarily defined.

Reply:
Okay, yes, it would be better than base.cfg.Thanks!

Reply:
Hi @vivianQizhu 
I checked the contents of windows.cfg and it seems to be used to save some configurations of the Windows VM. But base.cfg will save some examples of different parameters. Based on this, maybe base.cfg is a better place.What do you think?
FYI:https://github.com/avocado-framework/avocado-vt/blob/master/virttest/shared/cfg/guest-os/Windows.cfg

Reply:
@leidwang Or maybe here: https://github.com/avocado-framework/avocado-vt/blob/master/virttest/shared/cfg/virtio-win.cfg#L52